InteractionBase/CMakeLists.txt

```diff
@@ -192,4 +192,4 @@ add_library(InteractionBase SHARED
 	src/handlers/HMovableItem.cpp
 )
 
-envision_plugin(InteractionBase VisualizationBase FilePersistence)
\ No newline at end of file
+envision_plugin(InteractionBase VisualizationBase FilePersistence VersionControlUI)
```

Comment:
Why should interaction base depend on versioncontrol ui. This feels very broken.